sdb/commands/echo.py

```diff
@@ -24,6 +24,9 @@
 
 
 class Echo(sdb.Command):
+    """
+    ???
+    """
```

Comment:
`echo` is really the same as `address` but without the symbol-to-address resolution and the ability to get addresses from `drgn.Object`s passed through a pipe. It just forwards whatever it is passed from the pipe as is and adds to the pipe void* objects created by whatever address is passed as an argument.

In other words, `echo` in the beginning of the pipeline is like a crippled `address` command. But in the middle of a pipeline, it acts differently as it can add objects to an existing pipeline without bothering any pre-existing pipeline objects.

Reply:
OK, how does this sound:
```
Creates an object from the given address or addresses
```
I went back through the presentation and tests but didn't really see a good example

Reply:
Sounds good to me - you may want add that the type of this object is a `void *`